vars/manifest.yml

```diff
@@ -3,27 +3,36 @@ upstream_repos:
   openstack-helm-infra:
     src: https://git.openstack.org/openstack/openstack-helm-infra
     version: master
+    dest_subdir: openstack
   openstack-helm:
     src: https://git.openstack.org/openstack/openstack-helm
     version: master
+    dest_subdir: openstack
   openstack-helm-images:
     src: https://github.com/openstack/openstack-helm-images
     version: master
+    dest_subdir: openstack
   loci:
     src: https://github.com/openstack/loci
     version: master
-  airship-armada:
+    dest_subdir: openstack
+  armada:
     src: https://github.com/openstack/airship-armada
```

Comment:
this old link is now redirected to https://opendev.org/airship/. should we change to the new url?